configs/eslint-recommended.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+module.exports = {
+	rules: {
+		'no-nested-ternary': 'off',
+	},
+};
```

Comment:
I don't think this file name is a good name, but I don't have a good idea.

`conflicting-rules.js`?

Reply:
I got the file name from here: https://github.com/typescript-eslint/typescript-eslint/blob/8218055d6dfd94c9e6c8645848f981d9d51ce08c/packages/eslint-plugin/src/configs/eslint-recommended.ts. I think it made sense for them, because they were disabling **and enabling** a rules, so I think `conflicting-rules.js` would work.